static/src/javascripts/projects/common/modules/commercial/creatives/expandable-v2.js

```diff
@@ -48,63 +50,60 @@ define([
+        var topScroll = (topCusp === 'true') ?
+            ((scrollY + (viewportHeight * 0.4) - adSlotTop - adHeight) * 0.2) : 0;
 
         switch (this.params.backgroundImagePType) {
             case 'split':
+                this.scrollAmount = bottomScroll + topScroll + '%';
                 fastdom.write(function () {
-                    this.scrollAmount = bottomScroll + topScroll + '%';
                     $('.ad-exp--expand-scrolling-bg').css('background-repeat', 'no-repeat');
-                }.bind(this));
+                });
                 break;
             case 'fixed':
-                fastdom.read(function () {
-                    this.scrollAmount = (window.pageYOffset - this.$adSlot.offset().top) + 'px';
-                }.bind(this));
+                this.scrollAmount = (scrollY - adSlotTop) + 'px';
                 break;
             case 'parallax':
-                fastdom.read(function () {
-                    this.scrollAmount = ((window.pageYOffset - this.$adSlot.offset().top) * 0.15) + '%';
-                }.bind(this));
+                this.scrollAmount = ((scrollY - adSlotTop) * 0.15) + '%';
                 break;
         }
+
         fastdom.write(function () {
-            $('.ad-exp--expand-scrolling-bg').css('background-position', '50%' + this.scrollAmount);
-        }.bind(this));
+            $('.ad-exp--expand-scrolling-bg').css('background-position', '50%' + that.scrollAmount);
+        });
     };
 
     ExpandableV2.prototype.listener = function () {
-        fastdom.read(function () {
-            if ((window.pageYOffset + bonzo.viewport().height) > (this.$adSlot.offset().top + this.openedHeight)) {
-                // expires in 1 week
-                var week = 1000 * 60 * 60 * 24 * 7;
+        var that = this;
+        if ((window.pageYOffset + bonzo.viewport().height) > (this.$adSlot.offset().top + this.openedHeight)) {
+            // expires in 1 week
+            var week = 1000 * 60 * 60 * 24 * 7;
 
-                storage.local.set('gu.commercial.expandable.' + this.params.ecid, true, { expires: Date.now() + week });
-                fastdom.write(function () {
-                    this.$button.toggleClass('button-spin');
-                    $('.ad-exp__open-chevron').toggleClass('chevron-down');
-                    this.$ad.css('height', this.openedHeight);
-                    this.isClosed = false;
-                }.bind(this));
-                return true;
-            }
-        }.bind(this));
+            storage.local.set('gu.commercial.expandable.' + this.params.ecid, true, { expires: Date.now() + week });
+            fastdom.write(function () {
+                that.$button.toggleClass('button-spin');
+                $('.ad-exp__open-chevron').toggleClass('chevron-down');
+                that.$ad.css('height', that.openedHeight);
+                that.isClosed = false;
+            });
+            return true;
```

Comment:
I don't think it should. We can remove `return true` and test any expandable ad.